state.go

```diff
@@ -1078,7 +1079,7 @@ func (s *State) OnInterface(se *Session, i interface{}) (err error) {
 				oldCopy := *old
 				t.BeforeUpdate = &oldCopy
 			}
-			err = s.ChannelAdd(t.Channel)
```

Comment:
This error value is never used

Reply:
I think it's better to fix the shadowing rather than its consequences.